src/Shared-Libs/ManagedLoader/Datadog.AutoInstrumentation.ManagedLoader/internal/AssemblyResolveEventHandler.cs

```diff
@@ -54,7 +57,7 @@ internal IReadOnlyList<string> ManagedProductBinariesDirectories
 
         private static AssemblyName ParseAssemblyName(string fullAssemblyName)
         {
-            if (string.IsNullOrEmpty(fullAssemblyName))
+            if (String.IsNullOrEmpty(fullAssemblyName))

```

Comment:
is it a suggestion in that case ? (`String` over `string`) I hope it's not an error.